src/Persistence/Sql/Expression.php

```diff
@@ -493,6 +493,41 @@ public function __debugInfo(): array
         return $arr;
     }
 
+    protected function hasNativeNamedParamSupport(): bool
+    {
+        return true;
+    }
+
+    /**
+     * @param array{string, array<string, mixed>} $render
+     *
+     * @return array{string, array<string, mixed>}
+     *
+     * @internal
+     */
+    protected function updateRenderBeforeExecute(array $render): array
+    {
+        [$sql, $params] = $render = $render;
```

Comment:
probably bug here

Reply:
no bug, bug typo :)

Reply:
fixed and cheked all other code with `\$(\w+) = \$\1(?!\w|\(|->|\[| \?)` regex